src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,127 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double SHOULD_GROW_TIMES = 1.5;
+    private static final int STARTING_POSITION = 0;
```

Comment:
you can use numbers as 0 and 1 in your code; They are not considered magic numbers

Reply:
done